### ISSUE TITLE
NTR: Adrenal biopsy: adrenocortical hyperplasia (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9200]
New term request Adrenal biopsy: adrenocortical hyperplasia (elthat_101028015310):   UMLS: 0342494

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000152
- label: Adrenocortical hyperplasia
- parents: [Adrenocortical abnormality (HP:0000849)](https://hpo.jax.org/app/browse/term/HP:0000849)
- pmid: PMID:28707538
- def: A non-malignant growth, or enlargement, of the adrenal glands, specifically the cortex, although the cortex cannot be definitively identified by conventional imaging. Adrenal cortical hyperplasia is defined radiologically as a non-malignant growth, or enlargement, of the adrenal glands.
- comment: Controlled by the pituitary gland, the adrenal cortex drives critical processes, such as the production of cortisol, mineralocorticoid and sex hormones. Any disruption in the multiple enzymes and hormones involved in these pathways may cause serious or life-threatening symptoms, often associated with anatomical changes in the adrenal glands.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9200
- synonyms:
